maths/elu_activation.py

```diff
@@ -0,0 +1,39 @@
+"""
+Implements the Exponential Linear Unit or ELU function.
+
+The function takes a vector of K real numbers and a real number alpha as
+input and then applies the ELU function to each element of the vector.
+
+Script inspired from its corresponding Wikipedia article
+https://en.wikipedia.org/wiki/Rectifier_(neural_networks)
+"""
+
+import numpy as np
+
+
+def elu_activation(vector: np.ndarray, alpha: float) -> np.ndarray:
+    """
+         Implements the ELU activation function.
+         Parameters:
+             vector: np.array
+             alpha: float
+         return:
+         elu (np.array): The input numpy array after applying elu.
```

Comment:
Please do not repeat the data types from the function signature.  The problem with doing so is that the reader becomes confused if one changes and the other does not.  Either describe what these variables are for or remove these comments.